spec/rails/doc/smart/openapi.yaml

```diff
@@ -14,6 +14,19 @@ info:
 servers:
 - url: http://localhost:3000
 paths:
+  "/no-such-path":
+    get:
+      summary: no such api
+      parameters: []
+      responses:
+        '200':
+          description: dummy
+          content:
+            application/json:
+              schema:
+                type: number
+              example:
+                - 1
```

Comment:
Note) `/no-such-path` is added but does not appear in the`expected.json` because it is removed since it matched the pattern `1) paths.* (URL paths)`